templates/test/overview_result_table.html.ep

```diff
@@ -10,7 +10,8 @@
         </tr>
     </thead>
     <tbody>
-        % for my $config (sort keys %$type_results) {
+        % my @configs = sort { $a =~ s/@/ /r cmp $b =~ s/@/ /r } keys $type_results;
```

Comment:
wasnt't there something about keys on hash references being experimental? 

aka: I believe you are missing the % here.

Reply:
Just as I thought:

coolo@f84#~>perl -w -e 'my $a = { a => 1, b => 2 }; print keys $a;'
Experimental keys on scalar is now forbidden at -e line 1.
Type of arg 1 to keys must be hash or array (not private variable) at -e line 1, near "$a;"

Reply:
Wouldn't it make sense to add a perlcritic check for this?

Reply:
it's hard to perlcritic template code unfortunately :(

Reply:
fixed